src/sqlfluff/dialects/dialect_postgres.py

```diff
@@ -2331,7 +2331,7 @@ class InsertStatementSegment(BaseSegment):
     )
 
 
-@postgres_dialect.segment()
+@postgres_dialect.segment(replace=True)
```

Comment:
Can you remove the insert reference to this type in the Postgres StatementSegment?

Reply:
@WittierDinosaur Makes sense thanks, I've removed the insert reference now 👍